declarations.gms

```diff
@@ -1,14 +1,15 @@
 Parameters
-iCGI(allCy,YTIME)                           "Capital Goods Index (defined as CGI(Scenario)/CGI(Baseline))"
-iNPDL(SBS)                                  "Number of Polynomial Distribution Lags (PDL)"
-iFPDL(SBS,KPDL)                             "Polynomial Distribution Lags (PDL) Coefficients per subsector"
-iResDemSub(allCy,SBS,YTIME)                 "Residuals in total energy demand per subsector (1)"
+iCGI(allCy,YTIME)                            "Capital Goods Index (defined as CGI(Scenario)/CGI(Baseline))"
+iNPDL(SBS)                                   "Number of Polynomial Distribution Lags (PDL)"
+iFPDL(SBS,KPDL)                              "Polynomial Distribution Lags (PDL) Coefficients per subsector"
+iResDemSub(allCy,SBS,YTIME)                  "Residuals in total energy demand per subsector (1)"
 iLifChpPla(CHP)                              "Technical Lifetime for CHP plants (years)" /
 $ondelim
 $include "./iLifChpPla.csv"
 $offdelim
 /
 iCo2EmiFac(allCy,SBS,EF,YTIME)               "CO2 emission factors per subsector (kgCO2/kgoe fuel burned)"
+iUsfEnergy(allCy,SBS,EF,TEA,YTIME)           "Useful Energy Conversion Factor per subsector and technology ()"
```

Comment:
maybe add `(1)` as unit instead of `()`?

Reply:
Done https://github.com/giannou/OPEN-PROM/commit/29c9a16825f3454ce49c45f61ec1c1b6540fa33f